src/atoms/editor.ts

```diff
@@ -47,16 +47,13 @@ const baseActiveFileAtom = atomWithStorage(
   'guide:editor:activeFile',
   null as string | null
 );
-// Writable type needed if strictNullChecks is false
-// TODO: remove this when strictNullChecks is true
-export type Writable<T> = WritableAtom<T, any[], unknown>;
 export const branchAtom = atomWithStorage('guide:editor:branch', null);
-export const tokenAtom = atom(null) as Writable<string | null>;
+export const tokenAtom = atom(null);
```

Comment:
will this cause `tokenAtom` to lose the "string" typing? should we do something like `atom<string | null>(null)` instead?

Reply:
ah right, I missed that 😅